readme.md

```diff
@@ -8,16 +8,16 @@ to be executed in their programs.
 Currently must be installed from git
 
 ```
-npm i --save https://github.com/zazuko/rdf-native-loader-code
-npm i --save https://github.com/zazuko/rdf-native-loader
+npm i --save https://github.com/zazuko/rdf-loader-code
```

Comment:
Maybe this can already be changed to `npm I --save rdf-loader-code rdf-loaders-registry`?

Reply:
See comment about peer dependency.

Reply:
These comments are note related

I meant to change the git dependencies into package names

Reply:
Ups, sorry, didn't see the follow-up line in the conversation tab, so it looked like it's about installing 1 vs 2 packages. Yes, makes sense. Will fix it.

Reply:
While you're at it, the examples are also outdated...